src/Lumper.UI/ViewModels/MainWindowViewModel.cs

```diff
@@ -34,7 +34,32 @@ public MainWindowViewModel()
                 RxApp.MainThreadScheduler);
         }
     }
+    public async ValueTask UpdateCommand()
+    {
+        bool updateAvailable = await Lumper.UI.Updater.Updater.CheckForUpdate();
```

Comment:
Let's return the version, either by returning a nullable `Version` and checking for null, or keep the bool and use an out var. Will need to implement `ToString` on `Version` I guess.